src/components/teamBoxes/TeamTwoBox.tsx

```diff
@@ -5,50 +5,53 @@ import { child, get, onDisconnect, onValue, ref, set } from 'firebase/database';
+import { RootState } from '../../store';
 const TeamTwoBox = () => {
   const { roomId } = useParams();
 
-  const { playerId, username } = useSelector((state) => state.player);
+  const { playerId, username } = useSelector((state: RootState) => state.player);
   const teamTwoOperativesRef = ref(database, `rooms/${roomId}/team-2/operatives/`);
   const teamTwoSpymasterRef = ref(database, `rooms/${roomId}/team-2/spymaster/`);
   const teamOneRef = ref(database, `rooms/${roomId}/team-1/`);
-  const { teamTwoOperatives, teamTwoSpymaster } = useSelector((state) => state.teamTwo);
+  const { teamTwoOperatives, teamTwoSpymaster } = useSelector((state: RootState) => state.teamTwo);
   const playerOnTeamTwoOperativesRef = ref(database, `rooms/${roomId}/team-2/operatives/${playerId}`);
-  const teamTwoRemainingCards = useSelector((state) => state.game.team2RemainingCards);
+  const teamTwoRemainingCards = useSelector((state: RootState) => state.game.team2RemainingCards);
 
   const dispatch = useDispatch();
 
   // On click event for a player to be able to join team-2 team as a operative
   const joinTeamTwoOp = async () => {
-    //lines 23 - 35 are checking if the current player is already on a team
+    // lines 23 - 35 are checking if the current player is already on a team
     const teamOne = await get(teamOneRef);
     const teamOneOpsAndSpys = teamOne.val();
     let teamOneSpymaster;
     let teamOneOperatives;
-    //Grabbing team ones info
+    // Grabbing team ones info
     if (teamOneOpsAndSpys && teamOneOpsAndSpys.spymaster) {
       teamOneSpymaster = teamOneOpsAndSpys.spymaster;
     }
-    //Grabbing team ones info
+    // Grabbing team ones info
     if (teamOneOpsAndSpys && teamOneOpsAndSpys.operatives) {
       teamOneOperatives = Object.keys(teamOneOpsAndSpys.operatives);
     }
-    //If a player is on team 1, they cannot join this team
+    // If a player is on team 1, they cannot join this team
     if (
       (teamOneSpymaster && teamOneSpymaster.playerId === playerId) ||
       (teamOneOperatives && teamOneOperatives.includes(playerId))
     ) {
+      // tslint:disable-next-line:no-console
       console.log('Cannot join the other team!');
     } else {
-      //Here we want to check if a player is already a spymaster, so that they cannot join both
+      // Here we want to check if a player is already a spymaster, so that they cannot join both
       await get(teamTwoSpymasterRef).then((snapshot) => {
-        //If players already exist as team one spymasters:
+        // If players already exist as team one spymasters:
         if (snapshot.exists()) {
-          //'teamTwoSpymasters' sets the spymasers id's to an array
-          const teamTwoSpymaster = snapshot.val();
-          //Now we can check if the player is a spymaster, if they are, for now we just console log
-          if (teamTwoSpymaster.playerId === playerId) {
+          // 'teamTwoSpymasters' sets the spymasers id's to an array
```

Comment:
i forgot to take this comment out on my last PR - this 'array' comment is no longer accurate!

Reply:
done!